Readme.md

```diff
@@ -55,14 +68,17 @@ docker-compose exec php php /srv/mediawiki/maintenance/update.php
 
 Reload http://localhost:8080, you should see the main page.
 
-### Restore from backup
+### Option 2: Restore from backup
```

Comment:
We should move this option into the runbook.

Reply:
I have moved the command to the Runbook, but I would like to keep this link around, I think removing this from here will lead people to believe that this repository doesn't provide a script to restore backups.